concourse/parameters/test/deploy.yml

```diff
@@ -0,0 +1,6 @@
+govuk_infrastructure_branch: integration_pipelines
+concourse_deployer_role_arn: arn:aws:iam::430354129336:role/govuk-concourse-deployer
+workspace: fred
```

Comment:
Should this not be "default" workspace?

Reply:
This commit (9c21abf0d6aea8972db7562a82d71bb552545016), will be reversed, it is just for testing.